server.js

```diff
@@ -1,12 +1,21 @@
 const axios = require('axios');
-const dotenv = require('dotenv').config();
+const dotenv = require('dotenv').config({path: __dirname + '/.env'});
```

Comment:
This looks like a good solution for storing credentials. Let's make sure all of the team has set this up successfully and is getting the expected results in the test environment.

It might be good to explicitly add a test for this and the API calls that depend on it as well.